kubectl-rook-ceph.sh

```diff
@@ -344,6 +433,10 @@ function run_main_command() {
 : "${ROOK_CLUSTER_NAMESPACE:=rook-ceph}"
 : "${ROOK_OPERATOR_NAMESPACE:=$ROOK_CLUSTER_NAMESPACE}"
 : "${TOP_LEVEL_COMMAND:=kubectl}"
+: "${RESET:=\033[0m}"                    # For no color
+: "${ERROR:=\033[1;31mError $RESET}"     # \033[1;31m for Red color
```

Comment:
How does it look with a colon? Or maybe it's not necessary
```suggestion
: "${ERROR:=\033[1;31mError: $RESET}"     # \033[1;31m for Red color
```